pagopa/io-web-profile
# [IOPID-844] : Mixpanel UnLock Profile Flow

#  ⚠️ Depends on https://github.com/pagopa/io-web-profile/pull/50 ⚠️

## Short description
Adding Mixpanel Track Event for UnLock Profile Flow

## How to test
Run the project and check the console for the tracked event (mixpanel mocked), as specified in Figma, for the unlock profile flow.
